warehouse/oidc/services.py

```diff
@@ -161,6 +161,7 @@ def _get_key_for_token(self, token):
         return self.get_key(unverified_header["kid"])
 
     def verify_signature_only(self, token):
+        self.metrics.increment("warehouse.oidc.verify_signature_only.attempt")
```

Comment:
hmmm, any chance we know the provider at this point? would be good to tag so we know what providers are being used.

Reply:
Hmm, probably not directly -- we only test a JWT against a project's provider(s) once we're passed the signature verification, so plumbing the providers into this method would be inverting the dependency relationship.

The top level verification action is against a project, so perhaps a tag for the project name in `verify_for_project` is sufficient? WDYT?

Reply:
Thinking about this more: `verify_signature_only` should only ever be called from `verify_for_project`, so I can just remove these metrics and making the calling context more detailed.

Reply:
32cfd38bb43493cba21b23c133fe2617035baa40 and 63bce661e202f53ef704b21f353ebdb0d7e47689 address this.